source/views/components/html-view.js

```diff
@@ -8,7 +8,10 @@ export class HtmlView extends React.Component {
   props: {
     html: string,
     baseUrl?: ?string,
-    style?: ?string,
+    style?: ?any,
+    injectedJavaScript?: ?string,
+    javaScriptEnabled?: ?boolean,
+    onNavigationStateChange?: ?any,
```

Comment:
Instead of adding three new props, could you add one called, like, `autoAdjustHeight?: boolean`, please? And then just encapsulate the functionality into the HtmlView itself?

Or, like, make an AutoHeightHtmlView that wraps the HtmlView. Either way works, although I'd probably prefer the single prop.